runtests.py

```diff
@@ -826,8 +858,8 @@ def run_distutils(self, test_directory, module, workdir, incdir,
             ext_compile_flags = CFLAGS[:]
             compiler = COMPILER or sysconfig.get_config_var('CC')
 
-            if self.language == 'c' and compiler == 'gcc':
-                ext_compile_flags.extend(['-std=c89', '-pedantic'])
+            if self.language == 'c' and 'gcc' in compiler:
+                ext_compile_flags.append('-pedantic')
```

Comment:
Hmm, ok, I see. We currently require PY_LONG_LONG support in several places, which contradicts the intention of not requiring a "recent" C compiler. I'll respond to your mail on the mailing list.